src/Pages/geonwookim/Main/LeftSection/LeftSection.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import './LeftSection.scss';
+
+class LeftSection extends React.Component{
+  constructor(props){
+    super(props);
+      this.state={
+        inputText : "",
+        comments: [
+          {
+          text:"",
+        }
+        ],
+        id : '',
+        likeHeart: true,
+        likeCount: 1,
+      };
+    }
+
+    textChange = (e) =>{
+      this.setState({inputText: e.target.value})
+    }
+
+
+
+    add = () =>{
+      let texts = this.state.comments;
+      texts.push({text:'Wecode ' +this.state.inputText})
+
+      this.setState({comments: texts, inputText : "", })
+    }
```

Comment:
- 함수명 신경써주세요.
- this.state.comments에 담겨 있는 요소 하나 하나가 property를 잘 갖추도록 수정해주세요. 아래 코드 봐주세요.
- 해당 변수를 선언하고 push 하고 setState 해주기 보다는 아래와 같이 하면 좋습니다.

```suggestion
    add = () =>{
      const newComment = { userId: 'Wecode', content: this.state.inputText};
      this.setState({
          comments: [...this.state.comments, newComment ],
          inputText : ""
      })
    }
```